### PR TITLE
create functionality

### PR DESCRIPTION
**Pull request by ToddComte**

### CODE REVIEW

File: src/pages/CatNew.js [hunk 1]
```diff
@@ -1,10 +1,71 @@
+import { Button } from 'bootstrap'
```
Comment (sjproctor) [inline src/pages/CatNew.js]: Button should come from reactstrap as well

**Review by sjproctor — CHANGES_REQUESTED** [review]
Looks good, you are just missing your test coverge.

File: src/pages/CatNew.js (hunk 1 above)
Comment (ToddComte) [reply src/pages/CatNew.js]: ok i will take of it

**Review by sjproctor — APPROVED** [review]
Sweet!

**Pull request merged**